[2022-01-11 20:17:53,234] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2022-01-11 20:17:53,349] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2022-01-11 20:17:53,362] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:53,365] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-11 20:17:53,366] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:17:53,495] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_01> on 2022-01-08 00:00:00+00:00
[2022-01-11 20:17:53,519] {standard_task_runner.py:52} INFO - Started process 4744 to run task
[2022-01-11 20:17:53,549] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_01', 'scheduled__2022-01-08T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpur4db72f', '--error-file', '/tmp/tmpva8t1jlz']
[2022-01-11 20:17:53,611] {standard_task_runner.py:77} INFO - Job 42: Subtask task_01
[2022-01-11 20:17:54,370] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_01 scheduled__2022-01-08T00:00:00+00:00 [running]> on host airflowvm
[2022-01-11 20:17:55,049] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_01
AIRFLOW_CTX_EXECUTION_DATE=2022-01-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-08T00:00:00+00:00
[2022-01-11 20:17:55,050] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-11 20:17:55,050] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-01-11 20:17:55,099] {subprocess.py:85} INFO - Output:
[2022-01-11 20:17:58,287] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-11 20:17:58,673] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_01, execution_date=20220108T000000, start_date=20220111T201753, end_date=20220111T201758
[2022-01-11 20:17:59,038] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 20:17:59,874] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
